packages/zowe-explorer/src/trees/uss/USSAttributeView.ts

```diff
@@ -45,14 +43,11 @@ export class USSAttributeView extends WebView {
         switch (message.command) {
             case "refresh":
                 if (this.canUpdate) {
-                    this.onUpdateDisposable = this.ussNode.onUpdate(async (node) => {
-                        await this.attachTag(node);
-                        await this.panel.webview.postMessage({
-                            attributes: await this.ussNode.getAttributes(),
-                            name: node.fullPath,
-                            readonly: this.ussApi.updateAttributes == null,
-                        });
-                        this.onUpdateDisposable.dispose();
```

Comment:
I think we'll still need the `onUpdate` event listener. Once the node has been updated in the tree, the attributes in the webview are updated because this callback will be fired in `getChildren` ([see line 233 in ZoweUSSNode.ts](https://github.com/zowe/zowe-explorer-vscode/blob/b6dd3b50f69aee9660a2ae6c39b9a6646adb3e66/packages/zowe-explorer/src/trees/uss/ZoweUSSNode.ts#L233)). Then, on lines 53-57 in this file, the refresh is called on the tree node to get the new attributes.

It seems like removing this causes every other refresh call to work - for example, if I change the permissions in my terminal and then refresh in ZE, the first refresh does not show any changes, but the second one pulls the right permissions:
![ZE-EditAttributesRefresh](https://github.com/user-attachments/assets/ae80fd6d-97fe-49e1-a2a5-09aff54536c3)

I think we'll want to keep the `onUpdate` callback as it was working as expected in v2. It seems like maybe the `onUpdateEmitter` is not firing as expected, which is why the refresh doesn't update the webview with the current logic on `main`.

Reply:
I'm seeing the same behavior as Trae.
Also, the GIF shows that the `Apply changes` button is enabled the second time the refresh button is clicked, even though nothing the user (I) didn't make changes 😅